gril/grilunsol.c

```diff
@@ -39,6 +39,32 @@
 #include "common.h"
 #include "grilunsol.h"
 
+int g_ril_unsol_parse_connected(GRil *gril, const struct ril_msg *message)
+{
+	struct parcel rilp;
+	int size;
+	int version = RIL_VERSION_UNSPECIFIED;
+
+	DBG("");
```

Comment:
This line also appears in all other parser functions in this file. Shall I remove them as well?